ephios/core/templates/core/disposition/disposition.html

```diff
@@ -14,8 +14,11 @@
 {% endblock %}
 
 {% block content %}
-    <div class="page-header">
-        <h1>{{ shift }}</h1>
+    <div class="page-header text-break">
+        <h1>{{ shift.event }}</h1>
+        <h3 class="subtitle text-muted">
+            {{ shift.get_datetime_display }}
+        </h3>
```

Comment:
![grafik](https://user-images.githubusercontent.com/12702135/177427916-5343b71d-d84f-49cf-a46c-9de9c33321e6.png)
I think I'd like some more padding on the bottom here